src/hooks/useFollowScroll.ts

```
@@ -0,0 +1,70 @@
+  const targetPosition = useRef(0);
+  const currentPosition = useRef(0);
+  const [initialPosition, setInitialPosition] = useState<number | null>(null);
+  const deviceSize = useDeviceSize();
+
+  useEffect(() => {
+    if (deviceSize !== "desktop") return;
+
+    if (elementRef.current) {
+      try {
+        const initialTop = elementRef.current.getBoundingClientRect().top + window.scrollY;
+        if (isNaN(initialTop)) return;
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)